src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -257,6 +267,30 @@ public Long getTotalForSearch(TransactionSearchParams searchParams) {
         });
     }
 
+    public List<TransactionEntity> cursorTransactionSearch(TransactionSearchParams searchParams, ZonedDateTime startingAfterCreatedDate, Long startingAfterId) {
+        Long cursorPageSize = searchParams.getDisplaySize();
+        String cursorTemplate = "";
+        String searchTemplate = createSearchTemplate(searchParams.getFilterTemplatesWithParentTransactionSearch(), SEARCH_TRANSACTIONS_CURSOR);
+
+        if (startingAfterCreatedDate != null && startingAfterId != null) {
+            cursorTemplate = searchParams.getQueryMap().isEmpty() ? "WHERE " : "AND ";
+            cursorTemplate += "t.created_date < :startingAfterCreatedDate AND NOT (t.created_date = :startingAfterCreatedDate AND t.id < :startingAfterId) ";
```

Comment:
`t.created_date <= :startingAfterCreatedDate` ? or else we will exclude any transactions that haven't been included in previous page

Reply:
The second half of the query should take care of the `=`. It can be read as: 

Left hand side

* anything after our latest record

Right hand side: 

* anything that has the same (non-unique) date as our latest record but also has an ID that is less (as we've also ordered by ID this will be guaranteed)

Reply:
EDIT: update _simple_ query:
```
t.created_date <= :startingAfterCreatedDate AND NOT (t.created_date = :startingAfterCreatedDate AND t.id >= :startingAfterId)
```